### ISSUE TITLE
Prettier putting trailing comma after argument to dynamic import()

### ISSUE THREAD

**Issue by mrsharpoblunto** [prettier/prettier#2287]
I noticed that prettier will put a trailing comma after the argument to long dynamic import() calls e.g.

```
import(
  'myreallylongdynamicallyloadedmodulename',
).then(exports => {
});
```

According to flow (& from what I read in https://tc39.github.io/proposal-dynamic-import/) this is actually invalid syntax as import()'s are not the same as functions and don't support trailing commas after the argument.

**Comment by vjeux** [comment]
Thanks, i'll fix it.

**Comment by vjeux** [comment]
I just published 1.5.1 which fixes it, thanks for raising it!
